.github/workflows/tox-pytest.yml

```diff
@@ -10,19 +10,15 @@ on:
       - ready_for_review
```

Comment:
We should probably rename this workflow now that tox is no more.

Reply:
Yeah, I was just holding off since I think I have to change the `main` branch to do that right? And the name is insinuated into a bunch of different places.

Reply:
I want to rename the `update-lockfile` action too since that's a bit generic.